http4s/src/main/scala/com/avast/grpc/jsonbrige/http4s/Http4s.scala

```diff
@@ -59,13 +62,22 @@ object Http4s extends StrictLogging {
                         case Left(st) => mapStatus(st, configuration)
                       }
 
-                  case None => NotFound(s"Service '$serviceName' not found")
+                  case None =>
+                    val message = s"Attempt to POST non-existing service: $serviceName"
+                    logger.warn(message)
+                    NotFound(message)
                 }
 
-              case _ => BadRequest(s"Content-Type must be '${MediaType.application.json}', it's '$contentTypeValue'")
+              case Left(e) =>
+                val message = s"Content-Type must be '${MediaType.application.json}', it's '$contentTypeValue'"
+                logger.warn(message, e)
+                BadRequest(message)
             }
 
-          case _ => BadRequest(s"Content-Type must be '${MediaType.application.json}'")
+          case None =>
+            val message = s"Content-Type must be '${MediaType.application.json}'"
+            logger.warn(message)
```

Comment:
The same as above - too high logging level.

Reply:
lowered to `info`